src/commands/__tests__/undo-redo.ts

```diff
@@ -395,3 +524,66 @@ describe('grouping', () => {
     expect(exported).toEqual(expectedOutput)
   })
 })
+
+/******************************************************************
+ * MULTICURSOR - Undo/Redo.
+ ******************************************************************/
+
+describe('multicursor undo/redo', () => {
```

Comment:
You can remove `undo/redo` since this is a given since we are in the `undo-redo.ts` test file.